src/main/java/org/datadog/jmxfetch/reporter/Reporter.java

```diff
@@ -119,7 +119,15 @@ public void sendMetrics(List<Metric> metrics, String instanceName, boolean canon
                 }
                 sendMetricPoint(metricType, metricName, delta, tags);
 
-            } else { // The metric should be 'counter'
+            } else {
+                // `counter` and `rate` are equivalent and both accepted as valid.
+                // We continue accepting other cases for backward compatibility.
+                if (!"counter".equals(metricType) && !"rate".equals(metricType)) {
```

Comment:
IMO: this reads better as:
```
 if (!metricType.equals("counter") && !metricType.equals("rate")) {
    ...
}
```
But I just noticed we use this convention elsewhere in the file, so please feel free to ignore or fix both.

Reply:
Changed for this case and elsewhere.